mmv1/templates/terraform/custom_expand/billing_budget_budget_filter_labels.erb

```diff
@@ -0,0 +1,10 @@
+func expand<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d TerraformResourceData, config *Config) (map[string][]string, error) {
+	if v == nil {
+		return map[string][]string{}, nil
```

Comment:
What an unusual type for the API to expect, especially given it's restricted anyways. From the API docs: `Currently, multiple entries or multiple values per entry are not allowed.`